actions/image_partition_action.go

```diff
@@ -624,6 +642,16 @@ func (i *ImagePartitionAction) Verify(context *debos.DebosContext) error {
 		case "":
 			return fmt.Errorf("Partition %s missing fs type", p.Name)
 		}
+		if len(p.FSUUID) > 0 {
```

Comment:
this should be checked earlier ?

Reply:
We have moved the check a bit earlier but still within the verify function. Is that OK for you or did you mean something else?